curriculum/challenges/english/25-front-end-development/quiz-dom-manipulation-and-click-event-with-javascript/66edd07682767adff3a6231e.md

```diff
@@ -441,7 +441,7 @@ They both are `navigator` properties aliases.
 
 #### --text--
 
-Using the JavaScript Web animation API, how can you animate a "square" identified element to scale down to 0 in one second.
+Using the JavaScript Web animation API, how can you animate a `"square"` identified element to scale down to `0` in one second.
```

Comment:
```suggestion
Using the JavaScript Web animation API, how can you animate a `#square` element to scale down to `0` in one second.
```

```suggestion
Using the JavaScript Web animation API, how can you animate an element with the id of `"square"` to scale down to `0` in one second.
```

one of these two make more sense I think

Reply:
You are right, I personnaly like the second option, might be more clear to people not familiar with the `#` notation, I would even add "to scale *it* down".